adagucserverEC/CImageWarper.h

```diff
@@ -113,6 +113,7 @@ class CImageWarper{
     int reprojpoint(double &dfx,double &dfy);
     int reprojpoint(CPoint &p);
     int reprojpoint_inv(double &dfx,double &dfy);
+    int reprojpoint_inv_topx(double &dfx,double &dfy);
```

Comment:
Mij is het niet direct duidelijk wat de inv en de topx impliceren, misschien iets van commentaar om het te verduidelijken?

Reply:
Done